hail/python/hail/experimental/db.py

```diff
@@ -0,0 +1,64 @@
+from ..matrixtable import MatrixTable
+import requests
+import json
+import hail as hl
+
+class DB:
+    
+    _annotation_dataset_urls = None
+    
+    @staticmethod
+    def annotation_dataset_urls():
+        if DB._annotation_dataset_urls is None:
+            r = requests.get("https://www.googleapis.com/storage/v1/b/hail-common/o/annotationdb%2f1%2fannotation_db.json?alt=media")
+            j = r.json()
+            
+            DB._annotation_dataset_urls = {(x["name"], x.get("reference_genome")): x["path"] for x in j}
+            DB._geneDict_dataset_urls = { x["name"]: (x["gene_key"]) for x in j}
+        
+        return DB._annotation_dataset_urls, DB._geneDict_dataset_urls
+    
+    def annotate_rows_db(self,mt,*names):
+        """
+            Examples
+            --------
+            Annotates rows based on keyword implementation of annotation name.
+            The user can type in multiple annotation names when attaching to their datasets.
+            
+            >>> import hail as hl
+            >>> annotate_rows_db(mt,'DANN','CADD', '...', name='something_else')
+            >>> db = hl.annotation_database(config)
+            
+            # adds the vep, CADD, and gnomAD annotations to mt
+            >>> mt = db.annotate_rows_db(mt, 'vep', 'CADD', 'gnomAD')
+            ...
```

Comment:
Hail tests all of the examples in its docs and this one doesn't work. Let's change the whole block to just this:

```python
>>> db = hl.experimental.DB()
>>> mt = db.annotate_rows_db(mt, 'gnomad_lof_metrics')
```